src/lib/groovy/modsdotgroovy/EntrypointsBuilder.groovy

```diff
@@ -32,6 +37,54 @@ class EntrypointsBuilder {
         propertyMissing(name, args)
     }
 
+    /**
+     * Adds client entrypoints.
+     * @param entrypoints either the entrypoints to add
```

Comment:
The `@param` groovydocs seem wrong, says "either" but only specifies one thing. Probably forgot to remove that?

Reply:
Whoops

Reply:
Maybe include an example entrypoint value in the param?